thirdparty/restructuredtext_lint-1.3.0-cp37-none-any.whl.ABOUT

```diff
@@ -0,0 +1,15 @@
+about_resource: restructuredtext_lint-1.3.0-cp37-none-any.ABOUT
+name: restructuredtext_lint
```

Comment:
What is this new library for?
Also the notice URL may be rather https://github.com/twolfson/restructuredtext-lint/blob/1.3.0/UNLICENSE ?

Reply:
This library is same as https://github.com/nexB/scancode-toolkit/blob/develop/thirdparty/restructuredtext_lint-1.3.0-cp36-none-any.whl